src/modules/person/api/personenkontext.uc.ts

```diff
@@ -47,10 +53,32 @@ export class PersonenkontextUc {
             throw result.error;
         }
 
-        const personenkontexte: PersonenkontextResponse[] = result.value.map(
-            (personenkontext: PersonenkontextDo<true>) =>
-                this.mapper.map(personenkontext, PersonenkontextDo, PersonenkontextResponse),
+        const personenkontexte: PersonenkontextDto[] = result.value.map((personenkontext: PersonenkontextDo<true>) =>
+            this.mapper.map(personenkontext, PersonenkontextDo, PersonenkontextDto),
         );
         return personenkontexte;
     }
+
+    public async findById(dto: FindPersonenkontextByIdDto): Promise<PersonendatensatzDto> {
```

Comment:
name of the method, same issue as in the service.

Reply:
Done.